frontend/src/components/EOI/ExpressionOfInterestReadOnlyView.vue

```diff
@@ -1,221 +1,355 @@
+                  eoi[
+                    'iosas_incorporationcertificateissuedate@OData.Community.Display.V1.FormattedValue'
+                  ] || NULL_STRING
+                }}
+              </p>
+            </v-col>
+          </v-row>
+          <v-spacer />
+          <v-row>
+            <v-col cols="12" sm="12" md="6" xs="12">
+              <v-label>Certificate of Good Standing</v-label>
+              <div v-if="certificateOfGoodStandingDocument">
+                <v-icon aria-hidden="false" color="rgb(0, 51, 102)" size="20">
+                  mdi-file-document-check-outline
+                </v-icon>
+                {{ certificateOfGoodStandingDocument.iosas_file_name }}
+              </div>
+              <div v-else>{{ NULL_STRING }}</div>
+            </v-col>
+            <v-col cols="12" sm="12" md="6" xs="12">
+              <v-label>Certificate of Good Standing Issue Date</v-label>
+              <p>
+                {{
+                  eoi[
+                    'iosas_certificateofgoodstandingissuedate@OData.Community.Display.V1.FormattedValue'
+                  ] || NULL_STRING
+                }}
+              </p>
+            </v-col>
+          </v-row>
+          <v-label>Other</v-label>
+          <v-row>
+            <v-col cols="12" sm="12" md="4" xs="12">
+              <div
+                v-for="document in otherDocuments"
+                key="document.documentid"
+                v-if="otherDocuments"
+              >
+                <v-icon aria-hidden="false" color="rgb(0, 51, 102)" size="20">
+                  mdi-file-document-check-outline
+                </v-icon>
+                {{ document.iosas_file_name }}
+              </div>
+              <div v-else>{{ NULL_STRING }}</div>
+            </v-col>
+          </v-row>
+        </div>
+
+        <br />
+        <v-divider></v-divider>
+        <v-row>
+          <v-col cols="12" sm="12" md="12" xs="12">
+            <v-label>Additional Notes</v-label>
+            <p>{{ eoi.iosas_notes || NULL_STRING }}</p>
+          </v-col>
+        </v-row>
+      </div>
+    </div>
+  </v-container>
 </template>
 
 <script>
+import EOIFormHeader from './EOIFormHeader.vue';
 import { formatBooleanToYesNoString } from '../../utils/format';
 import { NULL_STRING } from '../../utils/constants';
 export default {
   name: 'ExpressionOfInterestReadOnlyView',
-  components: {},
+  components: {
+    EOIFormHeader,
+  },
   props: {
     eoi: {
       type: Object,
       required: true,
     },
+    draftStatusCode: {
+      type: Number,
+      required: true,
+    },
   },
   mounted() {},
   data() {
     return {
+      incorporationDocument: null,
+      certificateOfGoodStandingDocument: null,
+      otherDocuments: null,
       NULL_STRING,
     };
   },
+  created() {
+    if (this.eoi?.documents?.length > 0) {
+      this.incorporationDocument = this.eoi.documents.find(
+        ({ iosas_eoidocumenttype }) => iosas_eoidocumenttype === 100000000
```

Comment:
Can we use named/labeled const?  (100000000, 100000001)

Reply:
done!